index.js

```diff
@@ -179,6 +179,10 @@ function compareResultsAgainstGoldens(library) {
   } catch (err) {
     throw new Error(`Could not read expectedResults.json for ${library.name}:\n    ${err}`);
   }
+  // A constant, to make sure that if we add/remove any tests, that we add that test to all
+  // tested libraries.
+  const numberOfTests = 30;
```

Comment:
Is it worth making this constant computed (based off maybe the first library test)? No action required.

Reply:
My thinking lately has been that it's better to have this sort of constant checked in separately in source control, so that even if the other parts of the infra fail and would somehow pass with zero total tests then this will still fail

This isn't a constant that we'll update often, and when we do it'll be very deliberate, so it shouldn't be a maintenance burden either